tests/conditional_compilation/conftest.py

```diff
@@ -99,4 +99,50 @@ def artifacts(request):
 @pytest.fixture(scope="session")
 def openvino_root_dir(request):
     """Fixture function for command-line option."""
-    return request.config.getoption("openvino_ref")
+    return request.config.getoption("openvino_root_dir")
+
+
+@pytest.fixture(scope="session")
+def openvino_ref(request, openvino_root_dir, artifacts):
+    """Fixture function for command-line option.
+    Return path to root directory with installed OpenVINO.
+    If --openvino_ref command-line option is not specified firstly build and install
+    instrumented package with OpenVINO repository specified in --openvino_root_dir option.
+    """
+    openvino_ref_path = request.config.getoption("openvino_ref")
+    if openvino_ref_path:
+        return openvino_ref_path
+
+    build_dir = openvino_root_dir / "build_instrumented"
```

Comment:
Note `openvino_root_dir` is optional for now so that it can lead to failures in case of separate execution of each test